tests/integration/debug/cmd_debug_test.go

```diff
@@ -65,9 +107,6 @@ var _ = Describe("odo debug command serial tests", func() {
 			listenerStarted = true
 		}
 
-		debugInfoString := helper.CmdShouldPass("odo", "debug", "info", "--context", context)
-		Expect(debugInfoString).To(ContainSubstring(""))
```

Comment:
Debug info empty string is a key validation for command debug info output, so lets not remove this.

Reply:
Resolved :)